aws_securityhub/Dockerfile

```diff
@@ -20,7 +20,4 @@ RUN if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 # Install plugin
 RUN python setup.py build && python setup.py install
 
-# User to run plugin code. The two supported users are: root, nobody
```

Comment:
@mgorny-morecode  We want to define the user for each plugin. These changes will need to be reverted.